bigchaindb/common/schema/transaction.yaml

```diff
@@ -0,0 +1,256 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+id: "http://www.bigchaindb.com/schemas/transaction.json"
+type: object
+additionalProperties: false
+title: Transaction Schema
+description: |
+    This is the outer transaction wrapper. It contains the ID, version and the body of the transaction, which is also called `transaction`.
+required:
+- id
+- transaction
+- version
+properties:
+  id:
+    "$ref": "#/definitions/sha3_hexdigest"
+    description: |
+        A sha3 digest of the transaction. The ID is calculated by removing all
+        derived hashes and signatures from the transaction, serializing it to
+        JSON with keys in sorted order and then hashing the resulting string
+        with sha3.
+  transaction:
+    type: object
+    title: transaction
+    description: |
+        Body of the transaction.
+    additionalProperties: false
+    required:
+    - fulfillments
+    - conditions
+    - operation
+    - timestamp
+    - metadata
+    - asset
+    properties:
+      operation:
+        "$ref": "#/definitions/operation"
+      asset:
+        "$ref": "#/definitions/asset"
+      fulfillments:
+        type: array
+        title: "Fulfillments list"
+        description: |
+            Array of the fulfillments (inputs) of a transaction.
+
+            See: fulfillment_.
+        items:
+          "$ref": "#/definitions/fulfillment"
+      conditions:
+        type: array
+        description: |
+            Array of conditions (outputs) provided by this transaction.
+        items:
+          "$ref": "#/definitions/condition"
+      metadata:
+        "$ref": "#/definitions/metadata"
+      timestamp:
+        "$ref": "#/definitions/timestamp"
+  version:
+    type: integer
+    minimum: 1
+    maximum: 1
+    description: |
+        Transaction version identifier.
+definitions:
+  offset:
+    type: integer
+    minimum: 0
+  base58:
+    pattern: "[1-9a-zA-Z^OIl]{43,44}"
+    type: string
+  owners_list:
+    anyOf:
+    - type: array
+      items:
+        "$ref": "#/definitions/base58"
+    - type: 'null'
+  sha3_hexdigest:
+    pattern: "[0-9a-f]{64}"
+    type: string
+  uuid4:
+    pattern: "[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89ab][a-f0-9]{3}-[a-f0-9]{12}"
+    type: string
+    description: |
+        A `UUID <https://tools.ietf.org/html/rfc4122.html>`_
+        of type 4 (random).
+  operation:
+    type: string
+    description: |
+        Type of the transaction:
+
+        A `CREATE` transaction creates an asset in BigchainDB. This
```

Comment:
Sphinx is rendering these strings inside single backticks as italicized strings. For literal strings like "CREATE", maybe use double quotes around the string instead? An alternative is double backticks.